src/routes/home.jsx

```diff
@@ -1,7 +1,59 @@
+import { Modal } from "../components/modal/modal";
+
+import React, { useState, useEffect } from "react";
+
 export const Home = () => {
+
```

Comment:
The notifications are going to have their own route. Maybe go ahead and move the logic for the notification component to a separate file/route to avoid merge conflicts when the Home route gets flushed out.